www/templates/html/DefaultHomepage.tpl.php

```diff
@@ -61,8 +61,8 @@
     <div class="mh-featured">
       <a class="dcf-d-flex dcf-flex-col dcf-ai-center dcf-txt-decor-hover" href="<?php echo $baseUrl ?>manager/">
         <div class="mh-featured-icon dcf-d-flex dcf-ai-center dcf-jc-center dcf-h-10 dcf-w-10 dcf-circle unl-bg-scarlet">
-          <object type="image/svg+xml" data="<?php echo $baseUrl; ?>/templates/html/css/images/gear-icon.svg"  title="Manage Media Icon">
-            <img src="<?php echo $baseUrl; ?>/templates/html/css/images/gear-icon-white.png" alt="manage media">
+          <object type="image/svg+xml" data="<?php echo $baseUrl; ?>templates/html/css/images/gear-icon.svg"  title="Manage Media Icon">
+            <img src="<?php echo $baseUrl; ?>templates/html/css/images/gear-icon-white.png" alt="manage media">
```

Comment:
The SVG icons appear as white squares now. I assume it has to do with the updated file paths?

Reply:
@skoolbus39 - The path updates removed an extra `/` from the file path.  I don't see the issue.  Maybe do a hard refresh.  I rather not have the extra '/'.